css/teaser.css

```diff
@@ -85,6 +85,7 @@ body {
   height: 100px;
   margin: 0 auto;
   position: relative;
+  -webkit-user-select: none;
```

Comment:
Shouldn't we add prefixes for other browsers too? This won't work on Firefox or other major browsers.